Bindings/actuators.i

```diff
@@ -12,6 +12,7 @@
 %include <OpenSim/Actuators/Millard2012AccelerationMuscle.h>
 %include <OpenSim/Actuators/McKibbenActuator.h>
 %include <OpenSim/Actuators/DeGrooteFregly2016Muscle.h>
+%template (SetFunctionBasedPaths) OpenSim::Set<OpenSim::FunctionBasedPath>;
```

Comment:
To follow the naming convention of other sets --> `FuctionBasedPathSet`?

Reply:
In the past, naming for classes made up for swig interface started with Set while ones that existed in C++ used the C++ name which ended in Set, so we have the mix but if you feel strongly about it or plan to typedef in C++ I will make the change so let me know.

Reply:
Ah okay, I was mixing up name conventions. One more question, should the template name be pluralized, or just be `SetFunctionBasedPath`?

Reply:
I went back and forth on that and you'll see I initially had it without but looking at simulation.i that we most often use plurals
https://github.com/opensim-org/opensim-core/blob/7703592938979b54dc78ef4561781bbb7cc4994a/Bindings/simulation.i#L149